ui/app/i18n/registration/locale_fr.json

```diff
@@ -82,5 +82,11 @@
   "NO_FULFILMENT_MESSAGE": "Aucune observation capturée pour cette commande.",
   "SELECT_VALUE_FROM_AUTOCOMPLETE_DEFAULT_MESSAGE": "Veuillez sélectionner une valeur de auto complète",
   "REGISTRATION_ADDTIONAL_IDENTIFIERS": "Identifiants supplémentaires",
-  "PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER": "Téléphone"
+  "PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER": "Téléphone",
+  "HEART_FAILURE": "Heart Failure [FR]",
```

Comment:
@prashantsable1991 Wasn't it possible to add this translations as concept names for french locale instead?